staging/src/k8s.io/code-generator/cmd/go-to-protobuf/protobuf/generator.go

```diff
@@ -112,6 +112,9 @@ func isProtoable(seen map[*types.Type]bool, t *types.Type) bool {
 	case types.Map:
 		return isProtoable(seen, t.Key) && isProtoable(seen, t.Elem)
 	case types.Struct:
+		if len(t.Members) == 0 {
```

Comment:
can you create a separate PR for this? So we can get it in asap.

Reply:
Done. https://github.com/kubernetes/kubernetes/pull/55970.